apps/web/src/dialogs/command-palette/command-palette-dialog.tsx

```diff
@@ -69,10 +69,9 @@ export const CommandPaletteDialog = DialogManager.register(
 
     const commands = usePromise(async () => {
       select(0);
-      if (!defaultCommands.current)
-        defaultCommands.current = await getDefaultCommands();
+      defaultCommands.current = await getDefaultCommands();
```

Comment:
Why this change? We don't want to get default commands over and over again.

Reply:
We need the `getDefaultCommands` to re-run after a recent is removed from the user. Thanks for pointing this out, I realized we can add
```ts
defaultCommands.current = await getDefaultCommands()
``` 
in the click handler to fix this, instead of the above change

Reply:
Good idea.